edx-platform/pearson-vue-theme/lms/templates/user_authn/edx_ace/passwordreset/email/body.html

```diff
@@ -1,95 +1,60 @@
-                        background-position: right bottom;
-                        background-repeat: no-repeat;">
-
-                <h1 style="font-family:'Playfair Display', serif; font-weight: 700;">
-                    {% trans "Password Reset" %}
-                </h1>
-                <p style="padding: 0px;
-                          width: 50%;
-                          min-width: 300px;">
-                    {% blocktrans %}You're receiving this e-mail because you requested a password reset for your user account at {{ site_name }}.{% endblocktrans %}
-                </p>
-
+        <td>
+            <h1>
+                {% trans "Password reset" as tmsg %}{{ tmsg | force_escape }}
+            </h1>
+            <p style="color: rgba(0,0,0,.75);">
+                {% filter force_escape %}
+                {% blocktrans %}You're receiving this e-mail because you requested a password reset for your user account at {{ platform_name }}.{% endblocktrans %}
+                {% endfilter %}
+                <br />
+            </p>
 
-                {% if failed %}
-                <p style="padding: 0px;
-                          width: 50%;
-                          min-width: 300px;">
+            {% if failed %}
+                <p style="color: rgba(0,0,0,.75);">
+                    {% filter force_escape %}
                     {% blocktrans %}However, there is currently no user account associated with your email address: {{ email_address }}.{% endblocktrans %}
+                    {% endfilter %}
                     <br />
                 </p>
 
-                <p style="padding: 0px;
-                          width: 50%;
-                          min-width: 300px;">
-                    {% trans "If you did not request this change, you can ignore this email." %}
+                <p style="color: rgba(0,0,0,.75);">
+                    {% trans "If you did not request this change, you can ignore this email." as tmsg %}{{ tmsg | force_escape }}
                     <br />
                 </p>
             {% else %}
-                <p style="padding: 0px;
-                          width: 50%;
-                          min-width: 300px;">
-                    {% trans "If you didn't request this change, you can disregard this email - we have not yet reset your password." %}
+                <p style="color: rgba(0,0,0,.75);">
+                    {% trans "If you didn't request this change, you can disregard this email - we have not yet reset your password." as tmsg %}{{ tmsg | force_escape }}
                     <br />
                 </p>
-
-                 {% trans "Change my Password" as course_cta_text %}
-                {% include "ace_common/edx_ace/common/return_to_course_cta.html" with course_cta_text=course_cta_text course_cta_url=reset_link %}
-
-
-                 {% endif %}
-
-
-
-
-
-            </div>
+                {# xss-lint: disable=django-trans-missing-escape #}
```

Comment:
Why is this necessary? @luismorenolopera

Reply:
the next line contains forced_escaped, so it is necessary, https://edx.readthedocs.io/projects/edx-developer-guide/en/latest/preventing_xss/preventing_xss.html#django-trans-missing-escape